frontend/src/components/ManageSchema.vue

```diff
@@ -95,9 +145,20 @@
           </v-tab-item>
         </v-tabs-items>
       </v-container>
-
       <v-card-actions>
         <v-layout align-end justify-end>
+          <v-bpa-button
+            v-show="!isEdit && this.tab === 'schema-attributes'"
```

Comment:
Is there a better way to identify the current tab? Using hardcoded tab names seems a bit fragile.